{"name": "Нельзя войти с неверным паролем", "status": "passed", "start": 1715188980031, "stop": 1715188980928, "uuid": "842366fb-362d-4eff-b356-cc0dd5666a95", "historyId": "6dda1858ecf120e5d2e915ff5e9e17e8", "testCaseId": "6dda1858ecf120e5d2e915ff5e9e17e8", "fullName": "tests.test_login.TestLogin#test_login_with_invalid_password_not_completed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-9RKVCCS"}, {"name": "thread", "value": "6656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}